keyboards/wilba_tech/wt65_xt/keymaps/zunger/keymap.c

```diff
@@ -0,0 +1,659 @@
+  KC_ACCENT_START,
+  KC_CGRV = KC_ACCENT_START,  // Grave accent
+  KC_CAGU,  // Acute accent
+  KC_CDIA,  // Diaresis / umlaut / trema
+  KC_CCIR,  // Circumflex
+  KC_CCED,  // Cedilla
+  KC_CTIL,  // Tilde
+  KC_ACCENT_END,
+};
+
+enum layers_keymap {
+  _QWERTY = 0,
+
+  _FIRST_LANGUAGE_LAYER,
+  _YIDDISH = _FIRST_LANGUAGE_LAYER,
+  _SHIFTYIDDISH,
+  _GREEK,
+  _SHIFTGREEK,
+  _CADET,
+  _SHIFTCADET,
+  _LAST_LANGUAGE_LAYER,
+
+  // Function goes last.
+  _FUNCTION = _LAST_LANGUAGE_LAYER,
+};
+
+// We manage our OS mode internally, and store it in a static, rather than EEPROM, bit. That's
+// because it changes as we flip machines, and there's no good reason to wear out the memory.
+enum os_modes {
+  _WINDOWS = 0,
+  _MAC = 1,
+  _OS_MODES_MAX = 2,
+};
+static int os_mode = _MAC;
+
+// Key types matter for accent handling. If there's a pending accent request and another key is
+// pressed:
+// - If it's a normal key, we trigger all our magic accent handling.
+// - If it's a modifier key, we do nothing and let the accent request hold until the next keypress.
+// - If it's a special key, we drop the accent request but don't handle it.
+enum key_types {
+  _NORMAL_KEY = 0,
+  _MODIFIER_KEY = 1,
+  _SPECIAL_KEY = 2,
+};
+
+// msec to hold the platform key to trigger a switch
+#define PLATFORM_HOLD_DURATION 750
+
+// This is so that H(xxxx) has the same width as _______, which makes the grids more legible.
+#define H(x) UC(0x##x)
+#define MO_FN MO(_FUNCTION)
+#define KC_LLCK KC_LAYER_LOCK
+
+// Values for our OS-dependent keys, as arrays keyed by OS mode. Use Meet shortcuts on Mac, Teams on Windows
+const char *VC_MUTE_VALUES[_OS_MODES_MAX] = {SS_LCTL(SS_LSFT("m")), SS_LCMD("d")};
+const char *VC_HAND_VALUES[_OS_MODES_MAX] = {SS_LCTL(SS_LSFT("k")), SS_LCTL(SS_LCMD("h"))};
+const char *SCRNSHT_VALUES[_OS_MODES_MAX] = {SS_LGUI(SS_LSFT("s")), SS_LCMD(SS_LSFT("4"))};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[_QWERTY] = LAYOUT_all(
+		KC_ESC,      KC_LLCK,    KC_GRAVE,KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_HOME,
+		KC_PLATFORM, KC_MPLY,    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_PGUP,
+		KC_SCRNSHT,  KC_YIDDISH, KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,                    KC_PGDN,
+		KC_GREEK,    KC_CADET,   KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,                   KC_UP,   KC_END,
+		KC_VC_HAND,  KC_VC_MUTE, KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      MO_FN,            KC_LEFT, KC_DOWN, KC_RGHT),
+
+  /* The Greek and Cadet layers. Tab, backspace, the nav and modifier keys, and the control block
```

Comment:
It might be worth doing something like this:
https://github.com/qmk/qmk_firmware/blob/master/users/drashna/keyrecords/unicode.c#L100-L258

This replaces the typed characters with unicode, so you don't require extra layers

(eg, this is just a suggestion)

Reply:
Hah! I didn't realize that was already in QMK. Implementing exactly that functionality was on my TODO list. Man, QMK has _everything._ 

OK, I'm going to split these changes into two commits, because that one is going to be major surgery.